lib/warnings.pm

```diff
@@ -583,6 +583,9 @@ warnings - Perl pragma to control optional warnings
     use warnings;
     no warnings;
 
+    # Standard warnings are enabled by use v5.35 or above
+    use v5.35;
```

Comment:
Seems like a functional change to a Perl module? What's the purpose of this kind of change when the _ask_ was for documentation?

Reply:
This is documentation.

Reply:
The ./regen.pl script generates  lib/warnings.pm from regen/warnings.pl